src/css/card.css

```
@@ -0,0 +1,93 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+.card {
+  font-family: Roboto, Helvetica, sans-serif;
+  color: #060b35;
+}
+
+.card__wrapper {
+  max-width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product-image {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__product-title {
+  margin-bottom: 4px;
+  font-weight: 500;
+}
+
+.card__product-title,
+.card__product-price {
+  line-height: 18px;
+}
+
+.card__product-title,
+.price {
+  font-size: 12px;
+}
+
+.card__product-rating,
+.card__product-price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__product-rating {
+  margin-bottom: 24px;
+}
+
+.card__product-code {
+  line-height: 14px;
+}
+
+.card__product-code,
+.card__product-price {
+  margin-bottom: 16px;
+}
+
+.card__product-code,
+.reviews {
+  font-size: 10px;
+}
+
+.card__product-code,
+.price {
+  color: #616070;
+}
+
+.price-value,
+.card__button {
+  font-weight: 700;
+}
+
+.card__button {
+  display: block;
+  padding: 12px 0;
+
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-align: center;
+  color: #fff;
+
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+
+  transition: all 0.3s ease;
```

Good idea, I forgot about this thing!

---

You should define to what properties use transition.
For example
```suggestion
  transition: background-color 0.3s ease, color 0.3s ease;
```

---

Also, you could create variable for transition duration